bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -226,14 +221,17 @@ spec:
       - name: host-var-run-ovs
         hostPath:
           path: /var/run/openvswitch
+      - name: host-var-run-ovn-kubernetes
+        hostPath:
+          path: /var/run/ovn-kubernetes
       - name: host-sys
         hostPath:
           path: /sys
 
-      - name: host-opt-cni-bin
+      - name: host-cni-bin
         hostPath:
-          path: /opt/cni/bin
-      - name: host-etc-cni-netd
+          path: /var/lib/cni/bin
+      - name: host-cni-netd
         hostPath:
           path: /etc/kubernetes/cni/net.d
       - name: host-config-openvswitch
```

Comment:
While you're at it, can you move this to `/var/lib/openvswitch` like we do for openshift-sdn? We had selinux errors with trying to write to `/etc` before, and `/etc/origin` is no more.